pyproject.toml

```diff
@@ -36,16 +36,82 @@ where = ["."]
 [tool.setuptools.dynamic]
 dependencies = { file = ["requirements.txt"] }
 optional-dependencies = { dev = { file = ["requirements-dev.txt"] } }
-version = {attr = "luxonis_train.__version__"}
+version = { attr = "luxonis_train.__version__" }
 
 [tool.ruff]
 target-version = "py310"
 line-length = 79
 indent-width = 4
 
 [tool.ruff.lint]
-ignore = ["F403", "B028", "B905", "D1", "W191"]
-select = ["E4", "E7", "E9", "F", "W", "B", "I"]
+select = [
```

Comment:
Not all of the rules are enabled for now. The rest will be enabled with the nodes cleanup (which should be the last cleanup PR)